h2o-genmodel/src/main/java/hex/genmodel/ModelMojoFactory.java

```diff
@@ -90,7 +90,8 @@ public ModelMojoReader getMojoReader(String algo) {
         if (mmr != null) {
           return mmr;
         } else {
-          throw new IllegalStateException("Unsupported MOJO algorithm: " + algo);
+          throw new IllegalStateException("Algorithm `" + algo + "` is not supported by this version of h2o-genmodel. " +
+                  "If you are using algorithm implemented in an extension be sure to include a jar dependency of the extension (eg.: ai.h2o:h2o-genmodel-ext-" + algo.toLowerCase() + ")");
```

Comment:
Updated string:
"If you are using an algorithm implemented in an extension, be sure to include a jar dependency of the extension (eg.: ai.h2o:h2o-genmodel-ext-" + algo.toLowerCase() + ")");